proxy/envoy/config.go

```diff
@@ -106,11 +106,11 @@ func Generate(context *ProxyContext) *Config {
 			Clusters: clusters,
 			SDS: &SDS{
 				Cluster:        buildDiscoveryCluster(mesh.DiscoveryAddress, "sds"),
-				RefreshDelayMs: 1000,
+				RefreshDelayMs: (int)(mesh.DiscoveryRefreshDelay.Nanoseconds() / 1e6),
```

Comment:
nit: `int64(duration / time.Millisecond)` might be easier to understand. Duration is just int64 in nanoseconds (https://golang.org/pkg/time/#Duration)